### ISSUE TITLE
Alternative map sources

### ISSUE THREAD

**Issue by jaller94** [rastapasta/mapscii#128]
If I am understanding correctly, MAPSCII pulls its map data from webAPIs and renders it. Is it possible to host your own mapAPI, update the markers on the map server and then just use MAPSCII as a viewer?

_Originally posted by @rcmart3q in https://github.com/rastapasta/mapscii/issues/97#issuecomment-1404700492_

_Moved to its own issue because it was off-topic. Should be part of a in-tree documentation or wiki._

**Comment by jaller94** [comment]
Theoretically, it does support loading local mbtiles files.

```bash
npm install -g mapscii
npm install -g @mapbox/mbtiles
mapscii --source /home/rcmart3q/Documents/Vancouver.mbtiles
```

or another http(s) source (defaults to "http://mapscii.me/"):

```bash
npm install -g mapscii
mapscii --source http://localhost:8080/custom-map/
```

**Comment by rastapasta** [comment]
Correct, currently the vector tiles are served from https://mapscii.me/{z}/{x}/{y} - in case one uses a different source, the styles and [to-be-rendered layers (and their order)](https://github.com/rastapasta/mapscii/blob/master/src/Renderer.js#L295-L321) might need to be adapted
